### PR TITLE
Dm 2814

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2814
https://agile6.atlassian.net/browse/DM-2902

## Description - what does this code do?
1. Incorporates new designs for `Nominate an innovation` page.
2. Adds ability for user to subscribe to the DM newsletter on the homepage.

## Testing done - how did you test it/steps on how can another person can test it 
_**Nominate an Innovation**_
1. Visit `/nominate-an-innovation`
2. Immediately click on the "Send message" button and make sure the HTML5 required message appears. You can do this for each field if you so desire.
3. Fill out each field with the correct data, and then click "Send message" again.
4. Make sure you see a copy of the email, and that the data you passed in is correctly reflected in the email (note: you should see the "(Nominate)" pretext before the text you placed in the subject line input).
5. Go back to the nominate page and make sure you also see the success message at the top of the page.
6. Use your browser's inspection tool and make sure the mobile designs match the Figma mockups.

_**DM Newsletter Subscription**_
1. Visit '/'
2. Scroll down to the "Subscribe to the monthly newsletter" section.
3. Immediately try to click on the "Subscribe today" button and make sure you see the HTML5 required message.
4. Now add an email address and click the "Subscribe today" button.
5. Make sure you're taken to the GovDelivery site in a new browser window. You can choose to fill out the info or not.
6. Use your browser's inspection tool and make sure the mobile designs match the Figma mockups.

## Screenshots, Gifs, Videos from application (if applicable)

Screen Shot 2021-09-28 at 3 51 49 PM

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/xjaSwxytWsGGGCcc9f9Yfa/Handoff?node-id=103%3A69
https://www.figma.com/file/xjaSwxytWsGGGCcc9f9Yfa/Handoff?node-id=450%3A602

## Acceptance criteria
_**DM-2814**_
- Nominate a Practice link on homepage takes you to Nominate a practice page
- REMOVE “Nominate a practice” icon from homepage
- Include in email where user is coming from (“About” or “Nominate”)

_**DM-2902**_
- Design to add email address input to what we already have
- Make sure colors are consistent
- Build out subscribe 
- Stylize 

## Definition of done
- [X] Unit tests written (if applicable)
- [X] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -88,7 +88,8 @@
     get '/go-fish', action: 'go_fish', as: 'go-fish'
   end
 
-  get '/nominate-a-practice', controller: 'nominate_practices', action: 'index', as: 'nominate_a_practice'
+  get '/nominate-an-innovation', controller: 'nominate_practices', action: 'index', as: 'nominate_a_practice'
```
Comment (emyl3) [inline config/routes.rb:91]: did we want a redirect for the old path?

**Review by emyl3 — APPROVED** [review]
Left one comment/question -- everything else looked good to me! Great job, Josh!

File: config/routes.rb (hunk 1 above)
Comment (JoshingYou1) [reply config/routes.rb:91]: Good question! I'll ask Alex 😃

Comment (JoshingYou1) [reply config/routes.rb]: Okay, I added some code for a redirect!

**Review by emyl3 — APPROVED** [review]
Thank you! Looks good to me!

**Pull request merged**